scripts/set-sysctl.sh

```diff
@@ -0,0 +1,8 @@
+set -e
```

Comment:
Set `x` flag since this script will be used from `sudo` and it might be a good idea to show what is going on